src/excmds.ts

```diff
@@ -426,7 +378,7 @@ export async function guiset_quiet(rule: string, option: string) {
 //#background
```

Comment:
So //#background and //#content persist simply because some functions haven't gone into `{background,content}_lib/` yet or for some other reason?